nx-X11/programs/Xserver/Imakefile

```diff
@@ -248,7 +248,7 @@ NXAGENTOBJS = hw/nxagent/miinitext.o     \
               hw/nxagent/NXglxext.o      \
               hw/nxagent/NXmiexpose.o    \
               hw/nxagent/NXresource.o    \
-              hw/nxagent/NXdamage.o      \
+              hw/nxagent/damage.o        \
```

Comment:
This feels wrong (symlinks damage.[c|o] from DIX to DDX). Why not build damage.c at its original location?

Reply:
It is the same as with some other linked files: NXdamage.c did not have any code left after cleanup, but in dix/damage.c there are NXAGENT_SERVER ifdefs. That define is only set during nxagent compilation, but not during dix compilation.